docs/GoogleDrive-batchsink.md

```diff
@@ -4,23 +4,22 @@
 Description
 -----------
 Sink plugin saves files from the pipeline to Google Drive directory via Google Drive API.
-The minimal input schema contains only byte array with field name configured by **schemaBodyFieldName** property. 
-In this case files have rangomly generated 16-symbols names.
-Also file name can be retrieved from input scheme by configuring of **schemaNameFieldName** property.
 
 Properties
 ----------
 
-**clientId** OAuth2 client id.
+**File body field** The minimal input schema contains only byte array with field name configured by this property.
```

Comment:
The description should should this "what this property does".

Reply:
Also don't start with "the" or "a/an". Otherwise all descriptions would start like that.

Reply:
done